src/styles/loader.scss

```diff
@@ -0,0 +1,26 @@
+.formio-loader-wrapper {
```

Comment:
I'd prefer to use font-awesome and bootstrap styles for the loader. If we use custom styles then every user will have to import our css to make this work. If we use the generic styles they are more compatible.